clients/java/src/main/java/io/camunda/client/impl/search/SearchResponseMapper.java

```diff
@@ -56,12 +57,26 @@ public static SearchQueryResponse<ProcessInstance> toProcessInstanceSearchRespon
     return new SearchQueryResponseImpl<>(instances, page);
   }
 
-  public static SearchQueryResponse<UserTaskItem> toUserTaskSearchResponse(
+  public static SearchQueryResponse<UserTask> toUserTaskSearchResponse(
       final UserTaskSearchQueryResponse response) {
     final SearchQueryPageResponse pageResponse = response.getPage();
-    final SearchResponsePage page = toSearchResponsePage(pageResponse);
+    final SearchResponsePage page =
+        new SearchResponsePageImpl(
+            pageResponse.getTotalItems(),
+            pageResponse.getFirstSortValues(),
+            pageResponse.getLastSortValues());
```

Comment:
❓ Why are you not using the `toSearchResponsePage` method anymore?

Reply:
Accident :)

Reply:
Happens 👍 🙂